include/sentry.h

```diff
@@ -732,6 +735,23 @@ typedef void (*sentry_logger_function_t)(
 SENTRY_API void sentry_options_set_logger(
     sentry_options_t *opts, sentry_logger_function_t func, void *userdata);
 
+/**
+ * Enables or disables automatic session tracking.
+ *
+ * Automatic session tracking is enabled by default and is equivalent to calling
+ * `sentry_start_session` after startup. A running session will always be closed
+ * implicitly by `sentry_shutdown` if not done manually by calling
+ * `sentry_end_session`.
```

Comment:
Should we also explain that there can only be one concurrent session and that `sentry_start_session` closes the previous one?